wgsl/index.bs

```diff
@@ -1753,6 +1753,15 @@ The originating variable for a formal parameter of a function depends on the
 [=call site|call sites=] for the function.
 Different call sites may supply pointers into different originating variables.
 
+If a reference or pointer access is out of bounds, an <dfn noexport>invalid
+memory reference</dfn> is produced.
+[=Load Rule|Loads=] from an invalid reference return either:
+    * a value from any in bounds index for the reference
+    * the zero value for store type of the reference
+[=assignment statement|Stores=] to an invalid reference may either:
```

Comment:
According to the group consensus, this needs to also include the RBA behavior:
> Out-of-bounds writes may modify values within the memory range(s) bound to the buffer, but must not modify any other memory.

Which, again, effectively means **any** memory

Reply:
Same here